asmcomp/riscv/emit.mlp

```diff
@@ -243,10 +253,9 @@ let emit_instr env i =
       assert (env.f.fun_prologue_required);
       let n = frame_size env in
       emit_stack_adjustment (-n);
-      if n > 0 then cfi_adjust_cfa_offset n;
```

Comment:
Is there a reason for removing this line? It seems to do the right thing, even if CFI directives aren't fully right.

Reply:
Indeed. The line has been moved inside `emit_stack_adjustment` (as it was _missing_ from other places where this function was being used to adjust the stack pointer).